docs/api/safe-storage.md

```diff
@@ -0,0 +1,34 @@
+# safeStorage
+
+> Allows access to simple encryption and decryption of strings.
+
+Process: [Main](../glossary.md#main-process)
+
+Note that on Mac, access to the system Keychain is required and
+these calls can block the current thread to collect user input.
+The same is true for Linux, if a password management tool is available.
+
```

Comment:
Can we add a little more info here about what sort of attack this module is useful for protecting against? Why would a user want to use this API?